src/main.c

```diff
@@ -534,11 +534,29 @@ void dict(struct state *s)
 			}
 		} else if (strcmp(subfunc, "auth") == 0) {
 			if (peek_arg(&(s->cmd_list)) == NULL) {
-				printf("No username specified.\n");
-			} else {
+				char filename[] = "user.inf";
+				FILE*f;
+				int len;
+				rsp = exword_get_file(s->device, basename(filename), &buffer, &len);
```

Comment:
basename(filename) should not be necessary here. The basename of user.inf will still be user.inf.